code/model/modelanim.cpp

```diff
@@ -405,7 +405,13 @@ void model_anim_submodel_trigger_rotate(model_subsystem *psub, ship_subsys *ss)
 	Assert( psub != NULL );
 	Assert( ss != NULL );
 	Assert( psub->flags[Model::Subsystem_Flags::Triggered] );
-	Assert( ss->triggered_rotation_index >= 0 );
+	if (!psub || !ss || !psub->flags[Model::Subsystem_Flags::Triggered])
```

Comment:
This check isn't really needed since it is already covered by the `Assert`s from above.